src/logic/calculate.js

```diff
@@ -0,0 +1,20 @@
+import operate from './operate';
+/* eslint-disable no-param-reassign */
+
+const calculate = (calculator, buttonName) => {
+  let { total, next } = calculator;
+  const minusOne = -1;
+  if (buttonName === '+/-') {
+    total *= minusOne;
+    next *= minusOne;
+    calculator.total = total;
+    calculator.next = next;
```

Comment:
after destructuring on line 5, why refer to these properties again as 
- `calculator.total = total`;
- `calculator.next = next`;

??